cmd/utils/flags.go

```diff
@@ -182,6 +182,10 @@ var (
 		Name:  "lightkdf",
 		Usage: "Reduce key-derivation RAM & CPU usage at some expense of KDF strength",
 	}
+	BlockHashWhitelistsFlag = cli.StringSliceFlag{
```

Comment:
I don't think we need such a long name for the variable either. `WhitelistFlag` should be enough (note, singular too).

Reply:
We've also discussed this internally with @holiman and @fjl and decided that since all the flags that we have use comma separated value, we'd like this one to conform to the same style (even though you are right, this might allow setting the same flag multiple times).

Reply:
Ok, I've force pushed updates as b406a1c57 that changes to a `StringFlag`.